python/generate-invoice/main.py

```diff
@@ -0,0 +1,70 @@
+import json
+import base64
+from fpdf import FPDF
+
+class PDF(FPDF):
+    pass
+
+def error_message(res, message):
+    return res.json({"status": True, "message": message})
+
+def main(req, res):
+    currency = None
+    items = None
+    issuer = None
+    customer = None
+    vat = None
+    try:
+        payload = json.load(req.payload)
+        currency = payload["currency"]
+        items = payload["items"]
+        issuer = payload["issuer"]
+        customer = payload["customer"]
+        vat = payload["vat"]
+    except Exception as err:
+        print(err)
+        raise Exception("Payload is invalid.")
```

Comment:
Please make sure the function does now raise any exceptions. all exceptions should be caught and `error_message()` should be called before returning.